apm-lambda-extension/docs/aws-lambda-extension.asciidoc

```diff
@@ -0,0 +1,137 @@
+[[aws-lambda-extension]]
+= AWS Lambda Extension (Experimental)
+
+Elastic's APM Language Agents are able to instrument AWS Lambda functions via an AWS Lambda Extension.
+
+[[aws-lambda-arch]]
+== Extension Architecture
+
+Normally, during the execution of a Lambda function, there's only a single language process running in the AWS Lambda execution environment.  However, with an AWS Lambda Extension, Lambda users can run a _second_ process along side their main service/application process.
+
+// TODO: replace below with a diagram diagram
+
+[source,txt]
+----
+    [Lamnda Function/Service] -> [Lambda Extension] -> [APM Server]
+----
+
+By using a custom built AWS Lambda Extension, Elastic APM Agents can send data to a locally running Lambda Extension process, and that process will forward data on to APM Server.  The Lambda Extension ensures that any latency between the Lambda Function and the AWS Server instance will not cause latency in the Lambda Function/Service itself.
+
+[[aws-lambda-install]]
+== Installing the Lambda Extension
+
+Elastic offers an installer for adding the Lambda Extension to your Lambda functions.  This installer will
+
+1. Compile the Lambda Extension from source (written in go)
+2. Publish the Lambda Extension as a layer
+3. Configure a named Lambda Function with the just published layer
+4. Configure the required environmental variables.
```

Comment:
All other OL items don't have a period. Let's remove it from number four to be consistent.
```suggestion
4. Configure the required environmental variables
```